src/librustc_mir/util/borrowck_errors.rs

```diff
@@ -8,7 +8,7 @@ use std::fmt;
 #[derive(Copy, Clone, PartialEq, Eq, Debug)]
 pub enum Origin {
     Ast,
-    Mir,
+    Body,
```

Comment:
I think this use of `Mir` is correct

Reply:
Just because I'm curious - and I'm trying to learn rust - why do you think we should keep `Mir` here?

Reply:
The "MIR" term has been used in a double meaning up until this PR. It used to be for the entirety of `rustc::mir`, but also for `rustc::mir::Mir`. This PR changes `rustc::mir::Mir` to `rustc::mir::Body`, which makes it clear what that datatype actually is. The set of types in `rustc::mir` makes up the MIR,  and the `mir::Body` is one of these parts. 

So the `Origin` here should be `Mir` imo, even if the algorithm begins its processing at the `mir::Body`

Reply:
Got it. Thanks for explaining! :)